plugins/repository-s3/README.md

```diff
@@ -0,0 +1,21 @@
+# repository-s3
+
+The repository-s3 plugin enables the use of S3 as a place to store snapshots.
+
+## Testing
+
+### Unit Tests
+
+```
+./gradlew :plugins:repository-s3:test
+```
+
+### Integration Tests
+
+Integration tests require `amazon_s3_secret_key` and `amazon_s3_bucket` environment variables.
```

Comment:
We also need to provide `amazon_s3_access_key`, `amazon_s3_base_path` as mentioned in sample command below.

Reply:
Added.